CODEOWNERS

```diff
@@ -1,3 +1 @@
-*            @geeknoid @ozevren
-/appsignals  @louiscryan
-/filewatcher @lichuqiang
+* @istio/wg-test-and-release-maintainers
```

Comment:
Hmm. This is old, please rebase the PR.

Reply:
This is from the most recent commit on master (https://github.com/istio/pkg/commit/f287e58fcc78fab3757657fe4eaeb0885ea18882).  I'm not sure why it's showing up as part of my PR...